src/pybind/mgr/ssh/module.py

```diff
@@ -33,7 +33,8 @@
 
 DEFAULT_SSH_CONFIG = ('Host *\n'
                       'User root\n'
-                      'StrictHostKeyChecking no\n')
+                      'StrictHostKeyChecking no\n'
+                      'UserKnownHostsFile /tmp/ssh_known_hosts\n')
```

Comment:
Typically, you set this to `/dev/null` instead

```suggestion
                      'UserKnownHostsFile /dev/null\n')
```